index.bs

```diff
@@ -669,12 +678,13 @@ To <dfn>snapshot the lock state</dfn> for |origin| with |promise|:
 
 1. [=Assert=]: these steps are running on the [=lock task queue=].
 1. Let |pending| be a new [=list=].
-1. [=list/For each=] |request| of |origin|'s [=lock request queue=]:
-    1. Let |info| be a new {{LockInfo}} dictionary.
-    1. Set |info|'s {{LockInfo/name}} dictionary member to |request|'s [=lock request/name=].
-    1. Set |info|'s {{LockInfo/mode}} dictionary member to |request|'s [=lock request/mode=].
-    1. Set |info|'s {{LockInfo/clientId}} dictionary member to |request|'s [=lock request/clientId=].
-    1. [=list/Append=] |info| to |pending|.
+1. [=list/For each=] |queue| of |origin|'s [=lock request queue set=]:
```

Comment:
"lock request queue set" is no longer a thing. This will need to be "lock request queue map" and iterate using https://infra.spec.whatwg.org/#map-iterate